content/en/docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -32,32 +32,32 @@ DaemonSet has two update strategy types:
   before.
 * RollingUpdate: With `RollingUpdate` update strategy, after you update a
   DaemonSet template, old DaemonSet pods will be killed, and new DaemonSet pods
-  will be created automatically, in a controlled fashion. 
+  will be created automatically, in a controlled fashion.
 
 ## Performing a Rolling Update
 
 To enable the rolling update feature of a DaemonSet, you must set its
-`.spec.updateStrategy.type` to `RollingUpdate`. 
+`.spec.updateStrategy.type` to `RollingUpdate`.
 
-You may want to set `.spec.updateStrategy.rollingUpdate.maxUnavailable` (default
-to 1) and `.spec.minReadySeconds` (default to 0) as well.
+You may want to set [`.spec.updateStrategy.rollingUpdate.maxUnavailable`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#max-unavailable) (default
+to 1) and [`.spec.minReadySeconds`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) (default to 0) as well.
```

Comment:
actually, you don't have to use absolute paths here. You can use `/docs/concepts/.../#max-unavailable` as the href target, for example.